data/hai/unfettered duel jobs.txt

```diff
@@ -158,17 +158,55 @@ phrase "unfettered challenger defeat message"
 		`Here are your well deserved <payment>, captain. Next time I will not underestimate you`
 		`You must have had an edge! I'm in no condition to argue about giving you your payment of <payment>, however don't expect to see us again anytime soon`
 		`Here is your <payment>, captain, well earned`
-		`Whatever the case, you have earned my blade, and my respect, captain. You will be able to exchange it for <payment> the next time you land`
+		`Whatever the case, you have earned my weapon, and my respect, captain. I have transfered <payment> for you`
 		`We will analyze the fight to make sure you have not cheated. If that is the case, you will receive the <payment> shortly. If this was a fair fight, you may have gained my respect`
 	word
 		`."`
 	word
-		"	(Winning the duel has increased your reputation with the Unfettered. You are in no way allowed to do any more than plundering the ship.)"
+		"	(Winning the duel has slightly increased your reputation with the Unfettered. You are in no way allowed to do any more than plundering the ship.)"
 
 phrase "unfettered challenger captured"
 	word
 		`As soon as you captured the enemy challenger, you are hailed by the local unfettered authorities with a purple code priority transmission: "Never do that again. This is your first and final warning."`
 
+
+
+mission "Duel Rewards"
+	landing
+	source
+		attributes "unfettered"
+	to offer
+		has "unfettered challenger defeated"
+	on offer
+		conversation
+			`You open the hatch of your ship, only to discover a hai standing right in front of you, pointing pistols you have never seen before! This must be the captain you have just defeated.`
+			choice
+				`	(Wait for them to reveal their intentions.)`
+					goto explain
+				`	(Draw my weapon.)`
+			`	As you start to reach for your weapon the Unfettered gets closer to you, moving fast to put one of their pistols on your neck. "I like you," she says. "Ready to face danger, welcoming death! A shame you probably will not live that long with this attitude," she says before moving back slightly.`
+			label explain
+			`	She salutes you, pistol still in their hand, before presenting both weapons to you, as if making an offering. Seeing your apparent surprise, she starts speaking about the pistols, "Fascinating design, is it not? They come from the time of the raid. Even then, some of us decided it was better to take the initiative and attack them before they can, us! They developped this much more mobile version of the pulse rifle, and made it more deadly.`
+			`	You earned the right to have mine by defeating me in a duel. Of course we won't bother you every time you win one, but you'll be getting the pistols all the same, 3 by 3." She almost forcibly puts the pistols in your hand, and goes back to her ship, probably to supervise the repairs.`
```

Comment:
> 3 by 3

What does this mean?

Reply:
3 each time you land
~~that'd be clearer~~